bin/fuel-core/src/cli/run.rs

```diff
@@ -562,6 +562,7 @@ impl Command {
                 pool_limits,
                 heavy_work: pool_heavy_work_config,
                 service_channel_limits,
+                metrics: metrics.is_enabled(Module::TxPool),
```

Comment:
nit: We should probably rename `metrics` to `disabled_metrics` because that's what it is. I think `metrics` is a misleading name.

Reply:
done: e3f201d771

Reply:
The way we implemented it in the base PR is that:
1. we have a metrics config and CLI arg that manages "disabled metrics"
2. each module still has it's own "metrics: bool", which is initialized on startup by calling `.is_enabled(MODULE)`

This could be confusing at first, but my main motivation was to minimize the changes in all components.